src/web/rustdoc.rs

```diff
@@ -429,6 +429,18 @@ impl Handler for SharedResourceHandler {
 #[cfg(test)]
 mod test {
     use crate::test::*;
+    fn latest_version_redirect(path: &str, web: &TestFrontend) -> Result<String, failure::Error> {
+        use html5ever::tendril::TendrilSink;
+        let data = web.get(path).send()?.text()?;
+        let dom = kuchiki::parse_html().one(data);
+        for elems in dom.select("form ul li a.warn") {
```

Comment:
I'm a bit worried about this selector. If one day a second list item is added that is also `a.warn`, then this test might start to fail.
It would be nice if we could select this element using its text ("Go to latest version"), but... I imagine that would be pretty complicated to write ☹️ 

The effort to clean this up doesn't seem to be worth it for me, so I'm okay with keeping this as is.

Reply:
Since it's a test, we can fix it when it breaks without worrying about downtime, I don't think it's fragile enough to worry about until then.